spring-cloud-generator/generate-all.sh

```diff
@@ -13,6 +13,14 @@ cd ../
 PROJECT_VERSION=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout)
 cd spring-cloud-generator
 
+# When generate-one.sh fails, stores the captured stdout and stderr to a file
+# with the name of the client library
+# args 1 - library name;
+save_error_info () {
+  mkdir -p failed-library-generations
+  mv tmp-generate-one-output failed-library-generations/$1
```

Comment:
Nit: perhaps we could rename this folder to something like `preview-module-errors`?

Reply:
I think either name is fine, or something along the lines of `generation-error-logs`, as long as we don't intend to merge in the changes to main. It would be weird to have a failed/errors in our repo's main.
I imagine the process goes something like this:
- trigger generation from GithubAction
- Check on generated PR, look for anything in this folder. 
- In case there are errors, we probably don't want to merge in the pr. Will try to troubleshot or exclude and regenerate.
- If no errors occurred, this folder will not be populated to the PR.

Reply:
The action checks if any files are in `preview-module-errors`

Reply:
I will modify the folder name to a more understandable one once #1341 is merged